<launch>
    <!-- My Package launch file -->
    <node pkg="turtlebot3_move_real_robot" type="turtlebot3_sub_pub_action_server_RB.py" name="turtlebot3_move_action_server" output="screen"></node>
    <node pkg="turtlebot3_move_real_robot" type="turtlebot3_sub_pub_service_server_RB.py" name="turtlebot3_move_square_service" output="screen"></node>
    <node pkg="turtlebot3_move_real_robot" type="turtlebot3_sub_pub_RB.py" name="turtlebot3_move"  output="screen"></node>
</launch>

